scripts/generate-utility-data.ts

```diff
@@ -135,6 +139,8 @@ const OVERRIDES = new Map<string | number, string>([
   [27316, 'Stowe Electric Department'],
 ]);
 
+type UtilityMap = Map<string, { name: string }>;
+
```

Comment:
I'm not sure if there are style guide considerations advising the contrary, but to me if you're only using this in one method, I'd introduce it there instead.